downstream/modules/hub/proc-configuring-the-client-to-verify-signatures.adoc

```diff
@@ -62,11 +62,12 @@ name of your key file.
 +
 . Save and close the file.
 
-. Pull the file using podman, or your client of choice:
+.Verification
+* Pull the file using podman, or your client of choice:
```

Comment:
I think podman should be capitalized if it refers to the application? Tho it's not in the Supplemental Style Guide so I can't be sure 🤔

Reply:
Fixed.